### PR TITLE
Fix FSM rewiring RefillRequiredRound

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Fix FSM rewiring RefillRequiredRound

### CODE REVIEW

File: packages/valory/skills/trader_abci/composition.py [hunk 1]
```diff
@@ -76,6 +77,7 @@
     FinishedMarketManagerRound: SamplingRound,
     FailedMarketManagerRound: ResetAndPauseRound,
     FinishedDecisionMakerRound: RandomnessTransactionSubmissionRound,
+    RefillRequiredRound: ResetAndPauseRound,
```
Comment (angrybayblade) [inline packages/valory/skills/trader_abci/composition.py:80]: Before this change where were we transitioning to after the `RefillRequiredRound` round?

Comment (0xArdi) [reply packages/valory/skills/trader_abci/composition.py:80]: nowhere, it was missing

**Review by 0xArdi — APPROVED** [review]

**Review by angrybayblade — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — COMMENTED** [review]
Ok so I don't know if this will work. RefillRequiredRound(DegenerateRound) - not sure the framework allows moving through these rounds

**Review by DavidMinarsch — COMMENTED** [review]
Ah nm - this is just chaining and therefore replaced

**Comment by 0xArdi** [comment]
@DavidMinarsch its only an issue if not wired properly

**Review by Adamantios — COMMENTED** [review]
Why did we not delete the `RefillRequiredRound`? It has no purpose now.

**Pull request merged**